tests/gis_tests/distapp/tests.py

```diff
@@ -281,11 +284,17 @@ def test_geodetic_distance_lookups(self):
         line = GEOSGeometry('LINESTRING(144.9630 -37.8143,151.2607 -33.8870)', 4326)
         dist_qs = AustraliaCity.objects.filter(point__distance_lte=(line, D(km=100)))
 
-        self.assertEqual(9, dist_qs.count())
-        self.assertEqual(['Batemans Bay', 'Canberra', 'Hillsdale',
-                          'Melbourne', 'Mittagong', 'Shellharbour',
-                          'Sydney', 'Thirroul', 'Wollongong'],
-                         self.get_names(dist_qs))
+        if spatialite:
+            # SpatiaLite is less accurate and returns 102.8km for Batemans Bay.
+            self.assertEqual(['Canberra', 'Hillsdale',
```

Comment:
Adding `'Batemans Bay'` to the expected list if not spatialite rather than duplicating the assertion would be DRYer.